.github/workflows/firebase-hosting-pull-request.yml

```diff
@@ -0,0 +1,20 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on PR
+"on": pull_request
+jobs:
+  build_and_preview:
+    runs-on: ubuntu-latest
+    env:
+      REACT_APP_API_ROOT: https://corona-games.herokuapp.com
```

Comment:
We're deploying new client version, but still running it against old server 🙈 It will work fine as long as no game logic gets changed - to handle this we'd probably need to deploy backend as well, which will get expensive quite fast. Do we even need PR deployments?

Reply:
Or we could have automatic client deployments for branches and a single testing environment for server that could be manually deployed from PR when needed